core/src/test/java/ai/timefold/solver/core/impl/testdata/domain/clone/customcloner/TestdataScoreNotClonedSolution.java

```diff
@@ -26,7 +25,7 @@ public class TestdataScoreNotClonedSolution implements SolutionCloner<TestdataSc
     @ValueRangeProvider(id = "valueRange")
     @ProblemFactCollectionProperty
     public List<TestdataValue> valueRange() {
-        return Collections.singletonList(new TestdataValue("1"));
+        return List.of(new TestdataValue("1"), new TestdataValue("2"));
```

Comment:
Required to fix a test which otherwise wouldn't be able to reproduce the issue, as it would have terminated before it got there.